### PR TITLE
Restructure Providers and add GCP

### PR DESCRIPTION
**Pull request by mmcelroy91**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-87530

## Description of changes being made
PHASE 1 STRUCTURE and BASIC CONTENT ready to create/merge.  Additions to come as code is written.
 
Total work in progress that will take some time.... overhaul changing Choose Infrastructure section to Infrastructure Provider.  Also adding GCP section to Providers.  Also changing Install page to "Getting started."  Changing wording around "infrastructure" in paragraphs to include "provider" sometimes using both words depending on which infrastructure provider or setup it being documented.  

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4461.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/konvoy/2.3/getting-started/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: layout.pug
-navigationTitle: Install
-title: Install
+navigationTitle: getting-started
```
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.3/getting-started/index.md:3]: I think this is the "title" that will appear in the left-hand navigation bar, so maybe `Getting started` would fit here too.

**Review by gvasquezvargas — APPROVED** [review]
Just a tiny comment, everything else LGTM! I like this change very much!

File: pages/dkp/konvoy/2.3/getting-started/index.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ By combining a native Kubernetes cluster as its foundation with a default set of
 DKP provides a complete out-of-the-box solution for organizations that want to deploy production-ready
 Kubernetes.
 
-As an example, this Installation guide provides simplified instructions to get your DKP cluster up and running with minimal configuration requirements on an Amazon Web Services (AWS) public cloud instance. For information about installing on a different platform, see [Choose Infrastructure](../choose-infrastructure/). 
+As an example, this Installation guide provides simplified instructions to get your DKP cluster up and running with minimal configuration requirements on an Amazon Web Services (AWS) public cloud instance. For information about installing on a different platform, see [ Infrastructure providers](../choose-infrastructure/).
```
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.3/getting-started/index.md:27]:
```suggestion
As an example, this Installation guide provides simplified instructions to get your DKP cluster up and running with minimal configuration requirements on an Amazon Web Services (AWS) public cloud instance. For information about installing on a different platform, see [Infrastructure providers](../choose-infrastructure/). 
```

**Review by Colin1841 — APPROVED** [review]

**Comment by mmcelroy91** [comment]
CLOSED and started over... rebase did not work

**Pull request closed without merging**